### ISSUE TITLE
Pagination not rendering with routerMode: History

### ISSUE THREAD

**Issue by monibons** [imyelo/docsify-pagination#10]
If I set docsify's `routerMode` to `history` the pagination plugin fails to render. Is there a work around or fix for this?

**Comment by maplion** [comment]
My Pagination was rendering and then stopped for some unknown reason and I can't get it back.  I didn't set any routerMode settings, so I don't know if it is related.

**Comment by imyelo** [comment]
@maplion What is the error threw on the page?

**Comment by maplion** [comment]
There was none, it just refused to load that markdown.

**Comment by trusktr** [comment]
@maplion Are you using Vue in your markdown? If so, maybe this is the issue: https://github.com/imyelo/docsify-pagination/issues/16

**Comment by maplion** [comment]
Nope, no Vue.  It worked initially, then stopped working and I could never
get it to come back.

**Comment by rizdaprasetya** [comment]
I like this plugin and use it a lot. thanks authors, appreciate it 🎉

I also encounter the issue when Docsify set to `routerMode: 'history'`

I found quick (dirty) fix:
Using built script version of this plugin https://cdn.jsdelivr.net/npm/docsify-pagination@2.5.0/dist/docsify-pagination.js

Edit this line:
```
return decodeURIComponent(element.getAttribute('href').split('?')[0]) === decodeURIComponent('#' + path);
```
Replace with
```
return decodeURIComponent(element.getAttribute('href').split('?')[0]) === decodeURIComponent( window.$docsify.routerMode == "history" ? path : '#' + path);
```

It will be compatible for both `routerMode: 'history'` & `routerMode: 'hash'`
Hope it helps people who encounter same issue.

PS: I know there's [PR already open](https://github.com/imyelo/docsify-pagination/pull/11) addressing this issue, but it doesn't get merged yet. I'm desperately in need of the fix, so my trick above works for my needs.

**Comment by imyelo** [comment]
Fixed in #11 , and [v2.6.0](https://github.com/imyelo/docsify-pagination/releases/tag/v2.6.0) released

Please feel free to reopen

**Comment by rizdaprasetya** [comment]
Wonderful, thanks for the quick resolution
